rental.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'person'
+
+require_relative 'book'
+
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date)
+    @date = date
+
+    @person = personperson.rentals << self
+
+    @book = book
+
+    book.rentals << self
+  end
```

Comment:
- [ ] Since the rental belongs to a `book` and `person` objects the constructor needs to take in both of them as parameters and you need to implement proper assignment of their rentals.

  ```
  def initialize(date, book, person)
    @date = date
    
    @person = person
    person.rentals << self
    
    @book = book
    book.rentals << self
  end
  ```

Reply:
really appreciate